<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-Q0TAO2S" failures="0" tests="3" name="SauceDemo.SauceDemo.Asignacion5" time="6.991" errors="1" timestamp="2022-06-16T20:11:29 ART" skipped="2">
  <testcase classname="SauceDemo.SauceDemo.Asignacion5" name="abrirPagina" time="6.991">
    <error message="..\SauceDemo\Evidencias\Evidencias-SauceDemo.docx (El sistema no puede encontrar la ruta especificada)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: ..\SauceDemo\Evidencias\Evidencias-SauceDemo.docx (El sistema no puede encontrar la ruta especificada)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
at Utilidades.CapturaEvidencia.escribirTituloEnDocumento(CapturaEvidencia.java:71)
at SauceDemo.SauceDemo.Asignacion5.abrirPagina(Asignacion5.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:372)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- abrirPagina -->
  <system-out/>
  <testcase classname="SauceDemo.SauceDemo.Asignacion5" name="Login" time="0.000">
    <skipped/>
  </testcase> <!-- Login -->
  <system-out/>
  <testcase classname="SauceDemo.SauceDemo.Asignacion5" name="Preorden" time="0.000">
    <skipped/>
  </testcase> <!-- Preorden -->
  <system-out/>
</testsuite> <!-- SauceDemo.SauceDemo.Asignacion5 -->
